src/js/containers/covid19/awardSpendingAgency/AwardSpendingAgencyTableContainer.jsx

```diff
@@ -184,7 +184,9 @@ const AwardSpendingAgencyTableContainer = (props) => {
                     awardSpendingByAgencyChildRow.name = awardSpendingByAgencyChildRow.description;
                     if (query) {
                         awardSpendingByAgencyChildRow.name = replaceString(awardSpendingByAgencyChildRow.name, query, 'query-matched');
-                        expanded = awardSpendingByAgencyChildRow.name.length > 1;
+                        if (!expanded) {
```

Comment:
fwiw i hate this pattern where we assign markup to a mutable variable w/ `let`.

could aslo just define expanded w/ an `||` rather than adding this if statement.